cookbooks/continental_extension/continental_extension.prm

```diff
@@ -166,7 +214,7 @@ subsection Material model
   subsection Visco Plastic
 
     # Reference temperature and viscosity
-    set Reference temperature = 293
+    set Reference temperature = 273
```

Comment:
That's pretty cold :-)

Reply:
I actually don't recall why I made that change. Reasonable for the West Antartica Rift on consistent basis, but only 1 or 2 months a year in most rifts 😄. I'm actually wondering now how much that shift could affect dynamics long-term (e.g., changing the depth of the brittle-ductile transition)

Reply:
Ha, there's a research question for you: How does surface temperature affect rift behavior!